### PR TITLE
Onboarding functionality, some styling

### PR DESCRIPTION
**Pull request by chrismcanulty**
Further styling to come in separate JIRA ticket and PR

### CODE REVIEW

**Comment by chrismcanulty** [comment]
@wadesun18 Can you please help to review [eadad17](https://github.com/chrismcanulty/WorkoutGenerator/pull/6/commits/eadad175e2f319a9e95b29957b99089e0c8c1c79)?
Per tutorial at https://www.youtube.com/watch?v=z5oHeKseh1w
I have tried to use navigation.replace function to navigate to home/root screen but I am getting errors. Using the clearOnboarding function appears to work but I need to reload the app for the navigation to trigger

**Comment by chrismcanulty** [comment]
Most recent commits relate to WG-24 and WG-26

File: navigation/index.tsx [hunk 1]
```diff
@@ -19,33 +22,64 @@ export default function Navigation({
   colorScheme: ColorSchemeName;
 }) {
   return (
+    // <TailwindProvider>
```
Comment (wadesun18) [inline navigation/index.tsx]: Remove

File: navigation/index.tsx [hunk 2]
```diff
@@ -19,33 +22,64 @@ export default function Navigation({
...
       <RootNavigator />
     </NavigationContainer>
+    // </TailwindProvider>
   );
 }
 
 const Stack = createNativeStackNavigator();
 
 function RootNavigator() {
+  // set isOnboarding to false once onboarding has been completed
```
Comment (wadesun18) [inline navigation/index.tsx]: Do we need this comment?

File: navigation/index.tsx [hunk 3]
```diff
@@ -19,33 +22,64 @@ export default function Navigation({
...
+  const [isOnboarding, setIsOnboarding] = useState(true);
+  const checkOnboarding = async () => {
+    try {
+      const value = await AsyncStorage.getItem('@isOnboarding');
+      if (value !== null) {
```
Comment (wadesun18) [inline navigation/index.tsx]: Why is the name @isOnboarding?

File: screens/HomeScreen.tsx [hunk 4]
```diff
@@ -41,6 +46,9 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
       <Button onPress={() => navigation.navigate('MuscleGroup')}>
         <ButtonText>Generate</ButtonText>
       </Button>
+      <TouchableOpacity onPress={clearOnboarding}>
+        <Text>Clear onboarding</Text>
+      </TouchableOpacity>
```
Comment (wadesun18) [inline screens/HomeScreen.tsx:51]: There is to button to clear on-boarding? We should clear it when users finish on-boarding automatically

File: screens/OnboardingScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
+import {NativeStackHeaderProps} from '@react-navigation/native-stack';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {Image, SafeAreaView, Text, View} from 'react-native';
+import React from 'react';
+import styled from 'styled-components/native';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import Onboarding from 'react-native-onboarding-swiper';
+// import {useNavigation} from '@react-navigation/native';
```
Comment (wadesun18) [inline screens/OnboardingScreen.tsx]: Remove

File: App.tsx [hunk 6]
```diff
@@ -1,18 +1,27 @@
 import * as React from 'react';
+import {useEffect} from 'react';
 import {StatusBar} from 'react-native';
 import {useColorScheme} from 'react-native';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
 // import useCachedResources from './hooks/useCachedResources';
```
Comment (wadesun18) [inline App.tsx]: Remove

File: App.tsx [hunk 7]
```diff
@@ -1,18 +1,27 @@
...
 import Navigation from './navigation';
+import {TailwindProvider} from 'tailwindcss-react-native';
+import SplashScreen from 'react-native-splash-screen';
 
 export default function App() {
   // const isLoaded = useCachedResources();
```
Comment (wadesun18) [inline App.tsx]: Remove

**Review by wadesun18 — CHANGES_REQUESTED** [review]
Screenshots?

**Review by wadesun18 — APPROVED** [review]

**Pull request merged**